reservations/models/reservation.py

```diff
@@ -212,6 +210,14 @@ class Meta:
     def __str__(self) -> str:
         return f"{self.name} ({self.type})"
 
+    @property
+    def price_net(self) -> Decimal:
+        return round_decimal(self.price / (1 + self.tax_percentage_value / Decimal(100)), 6)
+
+    @property
+    def non_subsidised_price_net(self) -> Decimal:
+        return round_decimal(self.non_subsidised_price / (1 + self.tax_percentage_value / Decimal(100)), 6)
```

Comment:
Same as above with the (subsidised) net price, 2 decimal prices would be appropriate.